sql/migrations/error_detail_reports/000002_alter_error_detail_reports.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE error_detail_reports
+ADD COLUMN IF NOT EXISTS sample_event JSONB DEFAULT '{}'::JSONB,
```

Comment:
No a blocker, but an FYI:

We are considering  using TEXT, if we are not planning to query the json in the database, to avoid potential parsing/validation issues with POSTGRES.

```suggestion
ADD COLUMN IF NOT EXISTS sample_event TEXT DEFAULT '{}',
```

Reply:
> if we are not planning to query the json in the database

We are looking to query json in DB 😅